documentation/cybersecurity.workspace.adoc

```diff
@@ -0,0 +1,481 @@
+== Graphs for Cybersecurity
+:name: cybersecurity
+:long-name: Graphs for Cybersecurity
+:description: Cybersecurity, Active Directory environment auditing and analysis of possible attack paths using graph
+:icon: font
+:tags: Cybersecurity, ITSecurity, ActiveDirectory, Auditing, Attack Paths Analysis, AD Management
+:author: Neo4j
+:dump: https://github.com/neo4j-graph-examples/entity-resolution/tree/main/data
+:nodes: 953
+:relationships: 4858
+:img: img
```

Comment:
we might want to change this to https://guides.neo4j.com/auradb/cybersecurity/img